fastlane/Fastfile

```diff
@@ -427,6 +427,11 @@ platform :ios do
   lane :finalize_release do |options|
     UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
 
+    # Temporarily: we should remove after moving to Xcode 16 in definitive
```

Comment:
Would it make sense to automate this check so it's only really shown when we _know_ the `IMAGE_ID` is wrong?

```suggestion
sh("grep 'IMAGE_ID: xcode-15.4' #{PROJECT_ROOT_FOLDER}/.buildkite/release-builds.yml", error_callback: ->(result) {
      # Temporarily: we should remove after moving to Xcode 16 in definitive
      unless UI.confirm('Please ensure Xcode image is set to xcode-15.4 before moving forward, if it is, type y. If not, type n.')
        UI.user_error!('Before submitting the app to release please change the CI image to xcode-15.4')
      end
    })
```